buildmodel/commands.go

```diff
@@ -328,7 +328,7 @@ func SubmitUpdatePR(f *PRFlags) error {
 	runOrPanic(newGitCmd("commit", "-m", commitMessage))
 
 	// Push the commit.
-	args := []string{"push", *f.origin, b.PRBranchRefspec()}
+	args := []string{"push", auther.InsertAuth(*f.origin), b.PRBranchRefspec()}
```

Comment:
Missing auth for `runOrPanic(newGitCmd("fetch", "--no-tags", *f.to, b.PRBranchRefspec()))` and `runOrPanic(newGitCmd("fetch", "--no-tags", *f.origin, b.BaseBranchFetchRefspec()))` calls earlier?